src/components/Attachment/Attachment.js

```diff
@@ -51,7 +51,10 @@ export const isImageAttachment = (a) => {
 };
 
 export const isMediaAttachment = (a) => {
-  return a.mime_type && SUPPORTED_VIDEO_FORMATS.indexOf(a.mime_type) !== -1;
+  return (
+    (a.mime_type && SUPPORTED_VIDEO_FORMATS.indexOf(a.mime_type) !== -1) ||
+    a.type === 'video'
```

Comment:
Change looks good. Let's [add a test](https://github.com/GetStream/stream-chat-react/blob/b633f5c76087506ba34a56333888fabfdbdc6c72/src/components/Attachment/__tests__/Attachment.test.js#L101) for it?